# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1760090473 -7200
# Parent  a9771d98f322d482899b4d190ab587a073825bab
Misc: Switch off new Account Hub (too buggy) and fix Windows font rendering (bug 1946024).

Note that TB has now switched off the new Account Hub in bug 1992027.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -1508,4 +1508,9 @@ pref("calendar.dialogs.new.enabled", fal
 pref("mail.threadpane.listview", 0);
 
 // Row count for the cards view, currently bound to a range between 2 and 3.
 pref("mail.threadpane.cardsview.rowcount", 3);
+
+// Fit font rendering.
+pref("gfx.font_rendering.cleartype_params.force_gdi_classic_max_size", 15);
+pref("gfx.font_rendering.cleartype_params.force_gdi_classic_for_families",
+  "Arial,Consolas,Courier New,Microsoft Sans Serif,Segoe UI,Tahoma,Trebuchet MS,Verdana");
